modules/odo-dev-deploy.adoc

```diff
@@ -0,0 +1,28 @@
+// Module included in the following assemblies:
+//
+// * cli_reference/developer_cli_odo/understanding-odo.adoc
+
+:_content-type: CONCEPT
+[id="odo-dev-deploy_{context}"]
+= Develop and deploy
+
+The two most important commands in `odo` are `odo dev` and `odo deploy`. Sometimes, it can be hard to know which command to use.
```

Comment:
Consider rephrasing it so it's more user focused. What do you think of this rephrase? 

`...Learn when to use which command.` 
or
`...Identify when to use the correct command.` 

Something like that. What do you think?

Reply:
Agreed - used first option